app/models/folio/location.rb

```diff
@@ -7,8 +7,8 @@ module Folio
     def self.from_hash(dyn)
       new(
         id: dyn.fetch('id'),
-        campus: (Campus.new(**dyn.fetch('campus')) if dyn['campus']),
-        library: (Library.new(**dyn.fetch('library').symbolize_keys) if dyn['library']),
+        campus: (Campus.new(**dyn.fetch('campus').slice('id', 'code')) if dyn['campus']),
+        library: (Library.new(**dyn.fetch('library').slice('id', 'code', 'name').symbolize_keys) if dyn['library']),
```

Comment:
does `Campus` not need symbol keys?

Reply:
@cbeer do you know? @thatbudakguy Hasn't caused any problems in testing.